<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 236, 236);</string>
  </property>
  <widget class="QLabel" name="labelCMND">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 200);</string>
   </property>
   <property name="text">
    <string>CMND/CCCD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCMND">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(0,0,0,0);
border-color: rgba(46,82,101,100);</string>
   </property>
   <property name="placeholderText">
    <string>CMND/CCCD</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>127</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 200);</string>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(0,0,0,0);
border-color: rgba(46,82,101,100);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonSubmit
{
	border-radius:5px;
	color: rgba(255,255,255,255);
	background-color:rgba(85,98,112,255);
}

QPushButton#pushButtonSubmit:hover
{
	padding-left: 2px;
	padding-top: 2px;
	color: rgb(255, 0, 0);
}

QPushButton#pushButtonSubmit:pressed
{
	padding: 2px;
	color: rgba(255,255,255,255);
	background-color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurrentPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(0,0,0,0);
border-color: rgba(46,82,101,100);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Current Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>153</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 200);</string>
   </property>
   <property name="text">
    <string>Current Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChangePassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>219</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 200);</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelConfirmPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 200);</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditConfirmPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(0,0,0,0);
border-color: rgba(46,82,101,100);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
